### ISSUE TITLE
Multiple Language Support

### ISSUE THREAD

**Issue by just-Nob** [quaap/LaunchTime#20]
Would be nice to have multi language support.
There's still one pull request for Japanese, I'ld be willing to translate the resources to German, if desired...

**Comment by quaap** [comment]
Yup, translations are welcome!  But you might want to wait a bit, because I still consider this to be alpha/beta.  So the strings are changing pretty often as the app is developed: I'd hate to have a bunch of missing or incorrect translation entries and have to have everyone re-translate it again.

**Comment by just-Nob** [comment]
Ok, this sounds really reasonable and fair as well.
Just let me (and maybe others from other countries) know if you think the build is going to be stable enough, then I'm going to duo the German translation. :)

**Comment by just-Nob** [comment]
Seems the time is mature enough for translations! ;)
Anyway, I'm not sure how to do this. I've had a look into the Japanese translation's pull request / sources. Four me, it seems that naofum did the following:
1) forked sources,
2) added a folder "values-ja" for Japanese resource files,
3) copied the two xml-files "prefs.xml" and "strings.xml" into,
4) translated their content (strings),
5) did a pull request, so that you was able to merge them into your sources.

Is that correct, and is it the right way?
So I'd to fork, add a folder "values-de", do the translation and make a pull request?

Sorry for my questions, but I'm a newbe in Github as well as in Android programming.

**Comment by quaap** [comment]
Oh, yes.  I was planning on enlisting your help! :)   Thanks!  Yes, that seems correct. 

Things to keep in mind: 
* Anything with the tag translatable="false" doesn't need to be translated.
* Always choose the shortest translation, so that buttons etc will not grow too large or end up with clipped text.

Thanks again!

**Comment by just-Nob** [comment]
Ok, thx for your reply!

Yes, I know the problems with translated strings being to long for being completely displayed due to to little space... ;)

I guess the much more difficult part is to deal with Github (until now, my only experiences with version control systems were focused on SVN and - a long, long time ago - CVS) - anyway, I'll do my very best! :)

**Comment by quaap** [comment]
If you want, you can just translate the files directly and drop them back here. 
[values.zip](https://github.com/quaap/LaunchTime/files/799242/values.zip)

**Comment by quaap** [comment]
Hi!  I'd like to push the new version soon.  If it will be a while, let me know: I'm also waiting on a person to submit a Russian translation, so I can just do a translations-only release later.

**Comment by just-Nob** [comment]
Had made a branch, downloaded it to my Android mobile phone and am working on it these days.

I will make a pull request or - if I'm not able to upload the changes to Github - will put the files here directly.

Sorry for the delay, will be finished soon. I had and still have some family as well as job issues.

**Comment by just-Nob** [comment]
Hi Tom alias @quaap,
sorry for my inactivity the latest weeks; this was die to a bereavement as well as some further family issues.
Hope I will be back here soon. German translation was partly done before the bereavement, I have to check against LaunchTime's latest version to see wether the source string files have changed.
Please have a little patience, I will give you a pull request asap.

**Comment by Joppla** [comment]
I can help with translation to Dutch language.

**Comment by Crystal-RainSlide** [comment]
I'll take zh_CN
